Doc/reference/import.rst

```diff
@@ -539,25 +539,26 @@ the module.
    the module.  This name is used to uniquely identify the module in
    the import system.
 
```

Comment:
I don't see much meaningful difference for this file, so please leave this file alone.   (Let's avoid churn in the language reference.)  If you think there are improvements to be made here, let's do it in a separate BPO issue (and PR).  Thanks!

Reply:
Okay.